preprocess-train/preprocess-train.sh

```diff
@@ -15,10 +15,12 @@
 # limitations under the License.
 # ==============================================================================
 
-source activate rapids
+source activate merlin
```

Comment:
Should we remove this line? We don't have conda on the container anymore so I think this will fail.

Also, are we propagating errors up from this shell script to the calling process? I'm wondering if we should add a ```set -e``` flag at the beginning of this script (like https://www.davidpashley.com/articles/writing-robust-shell-scripts/#idm5413512)

Reply:
Removed